public/app/plugins/datasource/loki/components/LokiQueryFieldForm.tsx

```diff
@@ -164,7 +164,7 @@ export class LokiQueryFieldForm extends React.PureComponent<LokiQueryFieldFormPr
               {chooserText}
             </ButtonCascader>
           </div>
-          <div className="gf-form gf-form--grow flex-shrink-1">
+          <div className={'gf-form gf-form--grow flex-shrink-1 min-width-15 explore-input-margin'}>
```

Comment:
Just a small nitpick. I'd use this prop as a string like this
```suggestion
          <div className="gf-form gf-form--grow flex-shrink-1 min-width-15 explore-input-margin">
```

Reply:
Nice catch!